LoRaEngine/modules/LoRaWanNetworkSrvModule/Logger/LoRaConsoleLogger.cs

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
```

Comment:
Would be good to add `#nullable enable` for new code.

Reply:
Yeap - I'll add this to the template header in the editorconfig for consistency in the future.